Jint/Runtime/Debugger/DebugScope.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Jint.Native;
+
+namespace Jint.Runtime.Debugger
+{
+    /// <summary>
+    /// Dictionary of bindings for a single scope in the scope chain
+    /// </summary>
+    public class DebugScope : IReadOnlyDictionary<string, JsValue>
+    {
+        private IReadOnlyDictionary<string, JsValue> variables;
+
+        /// <summary>
+        /// The type of scope. Scope types are the same as defined by Chrome devtools protocol.
+        /// </summary>
+        public DebugScopeType ScopeType { get; }
+
+        internal DebugScope(DebugScopeType type, IReadOnlyDictionary<string, JsValue> variables)
+        {
+            ScopeType = type;
+            this.variables = variables;
+        }
+
+        #region IReadOnlyDictionary implementation
```

Comment:
ugh, regions..

Reply:
Oh, I never write them (or like them) either. 🤣 Except I explicitly added these two, just become *sometimes* I like easily getting boilerplate interface implementation out of the way. Sorry. 😉 They'll be gone.